### ISSUE TITLE
Add "Tree Feller" Plugin

### ISSUE THREAD

**Issue by brussels-sprout** [mc-ascendancy/ascendancy#241]
From **peaguy#9449** on Discord:

"Add 'ultimate timber.'"

**Comment by brussels-sprout** [comment]
This was not possible to add becuase it wouldn't recognise the custom trees but I found [this](https://www.spigotmc.org/resources/advanced-timber.65356) version that claims it does.

**Comment by connor7000** [comment]
Obviously, if we can configure it, there's no reason not to add it. Some of the trees on the server would take hours to chop down (nevermind not enabling leaf decay / fire tick being off).

It's a good idea. It is one we've considered implementing in the past, as you note @brussels-sprout. @mervinvb, could we look into the "Advanced Timber" plugin that sprout linked? Maybe we could attempt to see if it would be compatible/configurable/deployable!

**Comment by brussels-sprout** [comment]
@mervinvb said that plugin doesn't work but **snekky** brought up the idea of using a similar plugin made for ore veins and configuring it to work with wood that seems to somewhat work.
